src/core/directives/map.ts

```diff
@@ -247,6 +247,16 @@ export class AgmMap implements OnChanges, OnInit, OnDestroy {
   mapTypeId: 'roadmap'|'hybrid'|'satellite'|'terrain'|string = 'roadmap';
 
   /**
+   * This setting controls how gestures on the map are handled.
+   * Allowed values:
+   *     'cooperative' (Two-finger touch gestures pan and zoom the map. One-finger touch gestures are not handled by the map.)
+   *     'greedy'      (All touch gestures pan or zoom the map.)
+   *     'none'        (The map cannot be panned or zoomed by user gestures.)
+   *     'auto'        [default] (Gesture handling is either cooperative or greedy, depending on whether the page is scrollable or not.
+   */
+  gestureHandling: string = 'auto';
```

Comment:
I addressed the comments but I am now getting an issue with the webpack config when running the tests:
```
> karma start

12 03 2017 14:45:47.938:ERROR [preprocess]: Can not load "webpack"!
  WebpackOptionsValidationError: Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.entry should be one of these:
   object { <key>: non-empty string | [non-empty string] } | non-empty string | [non-empty string] | function
   The entry point(s) of the compilation.
   Details:
    * configuration.entry should NOT have less than 1 properties ({
        "keyword": "minProperties",
        "dataPath": ".entry",
        "schemaPath": "#/oneOf/0/minProperties",
        "params": {
          "limit": 1
        },
        "message": "should NOT have less than 1 properties",
        "schema": 1,
        "parentSchema": {
          "minProperties": 1,
          "additionalProperties": {
            "oneOf": [
              {
                "description": "The string is resolved to a module which is loaded upon startup.",
                "minLength": 1,
                "type": "string"
              },
              {
                "description": "All modules are loaded upon startup. The last one is exported.",
                "$ref": "#/definitions/common.nonEmptyArrayOfUniqueStringValues"
              }
            ]
          },
          "description": "Multiple entry bundles are created. The key is the chunk name. The value can be a string or an array.",
          "type": "object"
        },
        "data": {}
      }).
      object { <key>: non-empty string | [non-empty string] }
      Multiple entry bundles are created. The key is the chunk name. The value can be a string or an array.
    * configuration.entry should be a string.
    * configuration.entry should be an array:
      [non-empty string]
    * configuration.entry should be an instance of function
      function returning an entry object or a promise..
    at webpack (/Users/jimulle/Desktop/angular2-google-maps/node_modules/webpack/lib/webpack.js:19:9)
    at new Plugin (/Users/jimulle/Desktop/angular2-google-maps/node_modules/karma-webpack/lib/karma-webpack.js:63:18)
    at invoke (/Users/jimulle/Desktop/angular2-google-maps/node_modules/di/lib/injector.js:75:15)
    at Array.instantiate (/Users/jimulle/Desktop/angular2-google-maps/node_modules/di/lib/injector.js:59:20)
    at get (/Users/jimulle/Desktop/angular2-google-maps/node_modules/di/lib/injector.js:48:43)
    at /Users/jimulle/Desktop/angular2-google-maps/node_modules/di/lib/injector.js:71:14
    at Array.map (native)
    at Array.invoke (/Users/jimulle/Desktop/angular2-google-maps/node_modules/di/lib/injector.js:70:31)
    at Injector.get (/Users/jimulle/Desktop/angular2-google-maps/node_modules/di/lib/injector.js:48:43)
    at instantiatePreprocessor (/Users/jimulle/Desktop/angular2-google-maps/node_modules/karma/lib/preprocessor.js:55:20)
    at Array.forEach (native)
    at createPreprocessor (/Users/jimulle/Desktop/angular2-google-maps/node_modules/karma/lib/preprocessor.js:74:20)
    at Array.invoke (/Users/jimulle/Desktop/angular2-google-maps/node_modules/di/lib/injector.js:75:15)
    at get (/Users/jimulle/Desktop/angular2-google-maps/node_modules/di/lib/injector.js:48:43)
    at /Users/jimulle/Desktop/angular2-google-maps/node_modules/di/lib/injector.js:71:14
    at Array.map (native)
    at Injector.invoke (/Users/jimulle/Desktop/angular2-google-maps/node_modules/di/lib/injector.js:70:31)
    at Server.start (/Users/jimulle/Desktop/angular2-google-maps/node_modules/karma/lib/server.js:103:18)
    at Object.exports.run (/Users/jimulle/Desktop/angular2-google-maps/node_modules/karma/lib/cli.js:280:26)
    at Object.<anonymous> (/Users/jimulle/Desktop/angular2-google-maps/node_modules/karma/bin/karma:3:23)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)

START:
12 03 2017 14:45:47.974:INFO [karma]: Karma v1.3.0 server started at http://localhost:9876/
12 03 2017 14:45:47.975:INFO [launcher]: Launching browser Chrome with unlimited concurrency
12 03 2017 14:45:47.976:ERROR [karma]: Found 1 load error
```